src/components/CurrencyLogo/index.tsx

```diff
@@ -33,24 +35,23 @@ export default function CurrencyLogo({
   style?: React.CSSProperties
   className?: string
 }) {
+  const { chainId } = useWeb3React()
+  const nativeCurrencyLogo = NATIVE_CURRENCY_LOGO[(chainId as ChainId) || ChainId.MAINNET]
+
   const srcs: string[] = useMemo(() => {
-    if (currency === ETHER) return []
+    if (currency && Currency.isNative(currency) && !!nativeCurrencyLogo) return [nativeCurrencyLogo]
 
     if (currency instanceof Token) {
-      if (currency.name === 'DXdao') {
+      if (currency?.name === 'DXdao') {
```

Comment:
There's a question mark?  (this is a question :) )

Reply:
Ooops, my bad. That's indeed redundant, will remove it

Reply:
bd405cd2253c9651af25625903ff4546a35b94ea